pkgs/applications/networking/browsers/surf/default.nix

```
@@ -1,21 +1,22 @@
-{ lib, stdenv, fetchurl
+{ lib, stdenv, fetchgit
 , pkgconfig, wrapGAppsHook
-, glib, glib-networking, gsettings-desktop-schemas, gtk, libsoup, webkitgtk
+, glib, gcr, glib-networking, gsettings-desktop-schemas, gtk, libsoup, webkitgtk
 , xorg, dmenu, findutils, gnused, coreutils
 , patches ? null
 }:
 
 stdenv.mkDerivation rec {
   pname = "surf";
-  version = "2.0";
+  version = "2019-02-08-unstable";
```

```suggestion
  version = "unstable-2019-02-08";
```
See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming